packages/salesforcedx-vscode-apex-debugger/src/index.ts

```diff
@@ -268,23 +268,25 @@ export async function activate(context: vscode.ExtensionContext) {
     )
   );
 
-  console.log('Setting up ISV Debugger environment variables');
-  // register watcher for ISV authentication and setup default user for CLI
-  // this is done in core because it shares access to GlobalCliEnvironment with the commands
-  // (VS Code does not seem to allow sharing npm modules between extensions)
-  try {
-    registerIsvAuthWatcher(context);
-    console.log('Configured file watcher for .sfdx/sfdx-config.json');
-    await setupGlobalDefaultUserIsvAuth();
-  } catch (e) {
-    console.error(e);
-    vscode.window.showWarningMessage(
-      nls.localize('isv_debug_config_environment_error')
-    );
-  }
-
-  // Telemetry
   if (sfdxCoreExtension && sfdxCoreExtension.exports) {
+    if (sfdxCoreExtension.exports.isCLIInstalled()) {
```

Comment:
After fixing `salesforcedx-vscode-core` I noticed that `salesforcedx-vscode-apex-debugger` would also fail to activate when the cli is not present.

Reply:
Are these the only two packages that require the installation of the CLI?

Reply:
These are the only two packages that execute a cli command in their `activation` methods.

Reply:
Gotcha. Also I think you're missing the `showCLINotInstalledMessage()` call if it's not installed

Reply:
Are we calling CLI commands from extensions that don't require it in the activation specifically? Do we need to warn users in those cases too?

Reply:
@brpowell since `salesforcedx-vscode-apex-debugger` depends on `salesforcedx-vscode-core` I'm only using that in the core extension to only show the warning message once.

@allileong these two are the only extensions that calling the cli during activation. I think we can eventually move some of this out of activation but not immediately since some of those calls are being used to set the context used to show/hide commands and require a bigger refactor.
What type of warning are you thinking ?

Reply:
Sorry, my mistake. I looked in the code, and was thinking of a case that doesn't exist.